init-softlayer.html.md.erb

```diff
@@ -27,31 +27,36 @@ releases:{{ range .Releases }}
 
 resource_pools:
 - name: vms
-  network: private
+  network: default
   stemcell:
-    url: https://bosh.io/d/stemcells/bosh-softlayer-esxi-ubuntu-trusty-go_agent?v=3232.4
-    sha1: 1eda65c06cea25c3619eca0067a2d82dc100f0f4
-  cloud_properties: #TODO replace with SL specific cloud properties
-    cpu: 2
-    ram: 4_096
-    disk: 20_000
-  env:
-    bosh:
-      # c1oudc0w is a default password for vcap user
-      password: "$6$4gDD3aV0rdqlrKC$2axHCxGKIObs6tAmMTqYCspcdvQXh3JJcvWOY2WGb4SrdXtnCyNaWlrf3WEqvYR2MYizEGp3kMmbpwBC6jsHt0"
+    url: file://./light-bosh-stemcell-3169-softlayer-esxi-ubuntu-trusty-go_agent.tgz # <--- Replace with the one in bosh.io when it come to public
+  cloud_properties:
+    Domain: softlayer.com
+    VmNamePrefix: bosh-softlayer
+    EphemeralDiskSize: 100
+    StartCpus: 4
+    MaxMemory: 8192
+    Datacenter:
+      Name: SOFTLAYER-DATACENTER-NAME # <--- Replace with the datacenter (like lon02) for your environment
+    HourlyBillingFlag: true
+    PrimaryNetworkComponent:
+      NetworkVlan:
+          Id: SOFTLAYER-PUBLIC-VLAN # <--- Replace with the public vlan id for your environment
```

Comment:
please indent consistently with 2 spaces. also run this file through yaml parser.